cli/command/cluster/create.go

```diff
@@ -36,7 +36,7 @@ func NewCreateCommand(c cli.Interface) *cobra.Command {
 	flags.StringSlice("aws-parameter", []string{}, "Key-value pairs to pass through to the AWS CloudFormation template")
 	flags.String("aws-region", "", "The region to use when launching the instance")
 	flags.String("aws-stackname", "", "The name of the AWS stack that will be created")
-	flags.Bool("aws-sync", false, "If true, block until the command finishes (default: false)")
+	flags.Bool("aws-sync", true, "If true, block until the command finishes")
```

Comment:
Adding `(default: true)` to the help can be useful

Reply:
did you compile and run the help? It's displayed for me:
```      --aws-sync                       If true, block until the command finishes (default true)```

Reply:
yeah just checked 👍